controllers/topology/internal/extensions/patches/engine.go

```diff
@@ -248,7 +248,7 @@ func applyPatchesToRequest(ctx context.Context, req *api.GenerateRequest, resp *
 	log := tlog.LoggerFrom(ctx)
 
 	for _, patch := range resp.Items {
-		log = log.WithValues("templateRef", patch.TemplateRef)
+		log = log.WithValues("templateRef", patch.TemplateRef.String())
```

Comment:
We have to call the .String() func here explicitly because `WithValues` (and its underlying funcs) does not automatically use `String()` if the type implements the `Stringer` interface like `fmt.Sprintf` does